interface/countryconstructionsview.gui

```diff
@@ -6,6 +6,7 @@
 @QUEUED_TEXT_BOX_MAX_WIDTH = 20 # Prev 50
 @CLOSE_PROD_LINE_BUTTON_SCALE = 0.81 # Prev 1.0
 @PRIORITY_BUTTON_SCALE = 0.6 # Prev 1.0
+@MARKET_BUTTON_SCALE = 0.3 # Prev 1.0
```

Comment:
```suggestion
```
I don't think this is actually used, is it?

Reply:
correct